code/src/Summary.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Indentation is a little messy in this file - remember that opening and closing tags should be on the same level of indentation :)